### PR TITLE
st2ctl reload will also register all the enabled rules now.

### PR DESCRIPTION
**Pull request by samirsss**
Doc update for https://github.com/StackStorm/st2/issues/2861 change

### CODE REVIEW

**Comment by Kami** [comment]
As mentioned in the PR, please also update Upgrade Notes document.

Describe what has changed, why and how this affects the existing users (that's the general format we try to follow in Upgrade Notes).

File: docs/source/rules.rst [hunk 1]
```diff
@@ -235,7 +235,8 @@ The rules engine is able to interpolate variables by leveraging Jinja templating
 Managing Rules
 --------------
 
-To deploy a rule, use CLI ``st2 rule create ${PATH_TO_RULE}`` command, for example:
+To deploy a rule, use CLI ``st2 rule create ${PATH_TO_RULE}`` command, To reload all the rules,
+use ``st2ctl reload --register-actions`` for example:
```
Comment (Kami) [inline docs/source/rules.rst]: Shouldn't this be `st2ctl reload --register-rules` (not --register-actions)?

Comment (samirsss) [inline docs/source/rules.rst]: Fixed now.

**Comment by samirsss** [comment]
Updated the upgrade notes as well.

File: docs/source/upgrade_notes.rst [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 Upgrade Notes
 =============
+|st2| v1.6.1
+------------
+
+* st2ctl reload now also reloads all the actions. Prior to this release we used to register actions, aliases, sensors, triggers and configs. We now also register sensors.
```
Comment (Kami) [inline docs/source/upgrade_notes.rst]: @samirsss This reads a bit weird.

I assume it should say "st2ctl reload now also registers all the rules by default". And "Now rules are also registered by default".

Comment (Kami) [inline docs/source/upgrade_notes.rst]: Ignore that, I just pushed the change myself :)

Comment (lakshmi-kannan) [inline docs/source/upgrade_notes.rst]: Thanks @Kami. st2ctl reload now also reloads all the actions

**Pull request merged**